File: app/src/main/java/id/indosw/xlstojson/xlstojsonstring/Customer.java
package id.indosw.xlstojson.xlstojsonstring;

import org.jetbrains.annotations.NotNull;

public class Customer {

    private String id;
    private String name;
    private String address;
    private int age;

    public Customer() {
        super();
    }

    @SuppressWarnings("unused")
    public Customer(String id, String name, String address, int age) {
        super();
        this.id = id;
        this.name = name;
        this.address = address;
        this.age = age;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @SuppressWarnings("unused")
    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    @SuppressWarnings("unused")
    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    @NotNull
    @Override
    public String toString() {
        return "Customer [id=" + id + ", name=" + name + ", address=" + address + ", age=" + age + "]";
    }


}

File: app/src/main/java/id/indosw/xlstojson/kotlin/xlstojsonstring/Customer.kt
package id.indosw.xlstojson.kotlin.xlstojsonstring

class Customer {
    var id: String? = null
    var name: String? = null
    var address: String? = null
    var age = 0

    constructor() : super()
    constructor(id: String?, name: String?, address: String?, age: Int) : super() {
        this.id = id
        this.name = name
        this.address = address
        this.age = age
    }

    override fun toString(): String {
        return "Customer [id=$id, name=$name, address=$address, age=$age]"
    }
}